elytron/src/test/resources/org/wildfly/extension/elytron/credential-security-factories.xml

```diff
@@ -16,6 +16,7 @@
                                    server="false"
                                    obtain-kerberos-ticket="true"
                                    debug="true"
+                                   mechanisms="KRB5 KRB5LEGACY"
```

Comment:
Seems to me subsystem allowed values are still somehow static enumeration - not changed dynamically based on intern implementation. See for example:
* server-ssl-context.protocol
* syslog-audit-log.format
* configurable-sasl-server-factory.filters